auter

```diff
@@ -106,10 +107,22 @@ function prepare_updates() {
       # If check-update has an exit code of 100, updates are available.
       if [[ "${RC}" -eq 100 ]]; then
         sleep $(($RANDOM % ${MAXDELAY}))
-        PREPOUTPUT=$(${PACKAGE_MANAGER} ${PACKAGEMANAGEROPTIONS} update --downloadonly -y)
-        logit "INFO: Updates downloaded"
+        if [[ "${ONLYINSTALLFROMPREP}" == "yes" ]]; then
+          rm -f ${DOWNLOADDIR}/${CONFIGSET}/*.rpm
+          # DNF doesn't support downloaddir, so instead we download to the default
+          # location and copy out the way
+          if [[ "${PACKAGE_MANAGER}" == "dnf" ]]; then
+            find /var/cache/dnf -name *.rpm -exec rm -f {} \;
+            DOWNLOADOPTION=""
+          fi
+          DOWNLOADLOGMSG=" to ${DOWNLOADDIR}/${CONFIGSET}"
```

Comment:
Pedantism, but it seems weird to have "to " here.  Why not add it to the actual logit message?